src/App/Init.php

```diff
@@ -815,6 +815,12 @@
     return array_filter($config['data-providers']);
 });
 
+// HDX feature config
+$di->set('hxl.enabled', function () use ($di) {
```

Comment:
I really dislike this pattern. It makes feature flag way too complicated.
I'd prefer a more laravel like approach i.e. define a `Feature` facade and a `feature` helper. So you simple call `Feature::isEnabled('hxl')`.
TL;DR: Don't bother with injecting this